meta-ivi/recipes-support-ivi/ivi-logging/ivi-logging.inc

```diff
@@ -0,0 +1,23 @@
+DESCRIPTION = "ivi-logging"
+HOMEPAGE = "http://www.pelagicore.com"
+LICENSE = "MPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE.txt;md5=815ca599c9df247a0c7f619bab123dad"
+PR = "r0"
+
+DEPENDS = "glib-2.0"
+
+inherit cmake
+
+# Make DLT support optional
+# To enable, create a .bbappend with PACKAGECONFIG_append = "dlt"
+# in your project layer
+PACKAGECONFIG ??= ""
```

Comment:
Agreed.  Introducing ivi-logging was a GDP decision however.  I guess we should see which components in meta-ivi will be affected before the recipe is required here.  There are probably some, but let's take it in the right order.